src/CaptureClient/CompositeEventProcessorTest.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "CaptureClient/CaptureEventProcessor.h"
+
+namespace orbit_capture_client {
+
+class MockEventProcessor : public CaptureEventProcessor {
+ public:
+  MOCK_METHOD(void, ProcessEvent, (const orbit_grpc_protos::ClientCaptureEvent& event), (override));
+};
+
+TEST(CompositeEventProcessor, Smoke) {
```

Comment:
The name of the test doesn't reflect the name of the class (or vice versa).

Reply:
done